app/views/ops/_schedule_form_timer.html.haml

```diff
@@ -43,6 +43,7 @@
                    :name             => "start_date",
                    "ng-model"        => "scheduleModel.start_date",
                    "datepicker-init" => "",
+                   "miq-calendar"    => true,
```

Comment:
> We can use both 'datepicker-initand 'miq-calendar' at the same time, as long as we useDateobjects in the model andDate` is the preferred data type, am I correct?

If you don't mind occasional errors where we try to `update` unitialized datepicker, or doing `update` *and* reinitialize on every change, in nondeterministic order..

But.. I mean, it will *probably* work fine, maybe won't even show an error..